### DISCUSSION TITLE
Disk analyzer option

### DISCUSSION THREAD

**Discussion by SalahAdDin** [linuxdeepin/developer-center#3091] [Features Request & Ideas | 特性请求 & 头脑风暴]
By default, many OS with **Gnome** or **KDE** desktops come with a default disk analyser installed(**Boabab** for Gnome, for example).

But in deepin, we have a simple disk utility which does not offer this kind of utilities.

Also, the utility does not provide so much useful information or hellp to clean and manage the disk.

What do you think about it?

**Comment by meiyixiang** [comment]
Sorry, I don't quite understand what you mean. Could you be more specific?

**Reply by BLumia** [reply]
They are looking for a disk analyzer application like [Boabab](https://wiki.gnome.org/Apps/Baobab?action=show) and [Filelight](https://apps.kde.org/filelight/).

是希望 DDE 能提供一个类似 [Boabab](https://wiki.gnome.org/Apps/Baobab?action=show) 与 [Filelight](https://apps.kde.org/filelight/) 的磁盘占用情况分析软件

**Comment by meiyixiang** [comment]
Do you have only LVM capabilities?

**Comment by meiyixiang** [comment]
Hello to your suggestion, we have recorded this content needs to be evaluated
